docs/collections/_stream-tab.md

```diff
@@ -0,0 +1,28 @@
+Each collection in the GDN can be a stream. Collection streams use the WebSocket protocol to emit event messages for operations performed on the collection.
```

Comment:
```suggestion
Each collection in Macrometa GDN can be a stream. Collection streams use the WebSocket protocol to emit event messages for operations performed on the collection.
```

While it's technically correct, saying "the GDN" feels a bit weird to me.  I'm open to using it, but my instinct is to call it "Macrometa GDN" instead.